RP-Core/src/main/java/abilitiesV2/skills/abitilies/knowledge/gateofknowledge/wishes/ControlCreaturesWish.java

```diff
@@ -0,0 +1,291 @@
+                        getControlCreaturesEffect(livingCharacter)
+                                .apply(livingCharacter);
+                        getOrderCreaturesEffect(livingCharacter)
+                                .apply(livingCharacter);
+                    }, GateOfKnowledge.WISH_GRANT_RESPONSE_DELAY_SECONDS + 2);
+
+                }))
+                .denyWish(((livingCharacter, emissary) -> {
+                    runWithDelay(livingCharacter, () -> {
+                        emissary.speak(EmissaryMessageType.WISH_DENIED);
+                        emissary.getEmissaryWishlessBehaviour().getOnWrongWishAction().accept(livingCharacter, emissary);
+                    }, GateOfKnowledge.WISH_GRANT_RESPONSE_DELAY_SECONDS);
+                }))
+                .grantCondition(((livingCharacter, emissary) -> {
+                    boolean mastery = false;
+                    return mastery || RandomUtils.checkSuccess(50);
+                }));
+    }
+
+    private Effect getControlCreaturesEffect(LivingCharacter caster) {
+        SpecialCommandEffect takeControlEffect = new SpecialCommandEffect.Builder(SpecialCommandEffect.Type.CAN_B_COMMAND)
+                .applyActions(() -> {
+                    EffectMessage.send(caster,
+                            "You have received the power to control other creatures.",
+                            EffectMessageAudience.HOLDER);
+                })
+                .usageMessage(String.format("/b control - to take control of creatures in your %s radius.", TAKE_CONTROL_RADIUS))
+                .duration(DURATION_SECONDS)
+                .charges(TAKE_CONTROL_CHARGES)
+                .stackable(true)
+                .clearOn(true, false, true)
+                .useCondition(ControlCreaturesWish::isControlOrder)
+                .onUse(args -> {
+                    Location center = caster.getValidLivingEntity().getLocation();
+                    List<LivingCharacter> selectedCreatures = new CharacterSelector()
+                            .selectLivingCharactersInCube(center, TAKE_CONTROL_RADIUS)
+                            .stream()
+                            .filter(selected -> onlyMobs(selected, caster))
+                            .filter(selected -> !alreadyControlledByThisPlayer(selected, caster))
+                            .filter(selected -> TargetingUtils.isInClearPath(selected, caster, TAKE_CONTROL_RADIUS))
+                            .map(selected -> {
+                                shootControlMissiles(caster, selected);
+                                return selected;
+                            })
+                            .limit(MAX_CONTROLLED_PER_TAKE_CONTROL_ORDER)
+                            .collect(Collectors.toList());
+                    if (!selectedCreatures.isEmpty()) {
+                        EffectMessage.send(caster,
+                                "You attempt to take control of the nearby creatures.",
```

Comment:
If the character succeeds, why output "attempted"? 
Also, remove "the"
```suggestion
                                "You take control of nearby creatures.",
```

Reply:
Ye old message prior to refactor that has a separate msg for failure